bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -234,7 +234,7 @@ public BookieFormatCmd() {
             opts.addOption("f", "force", false,
                     "If [nonInteractive] is specified, then whether"
                             + " to force delete the old data without prompt..?");
-            opts.addOption("d", "deleteCookie", false, "Delete its cookie on zookeeper");
+            opts.addOption("d", "delete_cookie", false, "Delete its cookie on zookeeper");
```

Comment:
BC break here. Can you add a hidden option for deleteCookie, which behaves the same as delete_cookie but doesn't show up in the help?

Reply:
Thanks. I C, Since this change is to make a uniform between here and the help info in line 252, so would like to discard this change , and change line 252 into "deleteCookie". so they unified, and not break the old user behavior.